internal/bminventory/inventory.go

```diff
@@ -237,6 +261,96 @@ const nodeIgnitionFormat = `{
   }
 }`
 
+const configStaticIpsScript = `
```

Comment:
This is the script from Juan?

Reply:
yes, with minor changes, we don't need everything he does there